rules/game_process.tex

```diff
@@ -124,8 +124,42 @@ \subsubsection{Invalid Goal}
 In these cases a goal is not scored (that is, the goal is ruled invalid), and the game will proceed with a Goal Kick (\cf \cref{sec:free_kick}).
 The head referee should also advise why the goal is invalid, such as by calling ``Not indirect'', ``Played with hands'' or ``No own goal''.
 
-\subsubsection{Indirect Kick}
-\label{sec:indirect_kick}
+\subsubsection{Indirect Kick - Champions Cup}
+\label{sec:indirect_kick_champions}
+
+Both teams may only score a goal by making a second contact with the ball within is own team from any affected start or restart into \texttt{playing}, never mind who initiate the kick: 
```

Comment:
Both team may only score a goal through two distinct ball contacts involving different robots within its own team following any affected start or restart in the \texttt{playing} phase, regardless of the initiator of the set play/free kick.